Casks/calcservice.rb

```diff
@@ -7,5 +7,12 @@
   name "CalcService"
   homepage "https://www.devontechnologies.com/apps/freeware"
 
+  livecheck do
+    url "https://www.devontechnologies.com/support/download"
+    regex(%r{<td>CalcService</td><td>(\d+(?:\.\d+)*)</td>}i)
```

Comment:
It may be better to use the URL used for the download for regex.

```suggestion
    regex(/href=.*?v?(\d+(?:\.\d+)+)\/CalcService\.zip/i)
```

Reply:
It appears that the download `url` is not provided directly from `https://www.devontechnologies.com/support/download`.